aries-site/src/examples/components/tag/TagResource.js

```diff
@@ -37,57 +39,57 @@ const SectionDetails = ({ data }) => (
 );
 
 export const TagResource = () => (
-  <Box width={{ max: 'xxlarge' }} margin="auto" overflow="auto" fill>
-    <Box align="start" pad="medium">
-      <Button label="Devices" icon={<FormPrevious />} />
-    </Box>
-    <Box alignSelf="center" gap="large" pad="medium" flex={false}>
-      <Box
-        border={{ color: 'border-weak', side: 'bottom' }}
-        align="start"
-        direction="row"
-        justify="between"
-        pad={{ bottom: 'medium' }}
-        flex={false}
-      >
-        <Box>
-          <Heading size="small" margin="none">
-            Store C-3P0
-          </Heading>
-          <Text size="large">Aruba 530 32930F</Text>
+  <Page kind="narrow">
+    <PageContent gap="small">
+      <Button alignSelf="start" label="Devices" icon={<FormPrevious />} />
+      <Box alignSelf="center" gap="large" flex={false}>
```

Comment:
```suggestion
      <Box gap="large" flex={false}>
```
Can we remove this? seems like narrow page should take care of width and centering.